app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
     android:versionName="1.17" >
 
     <uses-sdk
-        android:minSdkVersion="9"
+        android:minSdkVersion="11"
```

Comment:
I see, thanks!
Would you mind removing that from the SDK then? Better not be redundant :-)

On Mon, Jul 11, 2016 at 5:29 PM, Josephine Lim <notifications@github.com>
wrote:

> In app/src/main/AndroidManifest.xml
> <https://github.com/nicolas-raoul/apps-android-commons/pull/169#discussion_r70218856>
> :
>
> > @@ -4,7 +4,7 @@
> >      android:versionName="1.17" >
> >
> >      <uses-sdk
> > -        android:minSdkVersion="9"
> > +        android:minSdkVersion="11"
>
> The Gradle build file says minSdkVersion 11, and the value specified in
> the Gradle build file overrides the manifest. Actually my IDE suggests
> removing the target SDK and min SDK lines entirely from the manifest, since
> the Gradle build file takes precedence so they are irrelevant. But I
> figured that preventing confusion by making the manifest values match the
> Gradle values would be sufficient.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nicolas-raoul/apps-android-commons/pull/169/files/2bace601299dcc4e3c1055e85d803e59a68badd4#r70218856>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGFBjjvEusqOjSKYHd4yDr5Bmamb0zsks5qUf7-gaJpZM4JJIU5>
> .
>